src/viking.js

```diff
@@ -1,11 +1,102 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+
+    attack() {
+        return this.strength = this.strength;
```

Comment:
why the assignment to itself? 
you could just do return this.strength :man_shrugging: and it would do the exact same and be shorter :)